task/backend/task.go

```diff
@@ -0,0 +1,31 @@
+package backend
+
+import (
+	"context"
+	"time"
+
+	"github.com/influxdata/influxdb"
+)
+
+// TaskControlService is a task service with control functions.
```

Comment:
This doesn't really inform the reader much about the type. Maybe something like:

> TaskControlService is a low-level controller interface, intended to be passed to
> task executors and schedulers, which allows creation, completion, and status updates of runs.

Reply:
Thanks that's very clear